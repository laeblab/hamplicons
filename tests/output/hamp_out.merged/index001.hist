151	1392
152	1961
153	5328
154	3724
155	26914
156	66168
157	1606
158	444
159	218
160	310
161	187
162	109
163	159
164	104
165	109
166	85
167	74
168	105
169	96
170	61
171	75
172	17
173	24
174	60
175	33
176	56
177	12
178	40
179	21
180	31
181	10
182	3
183	4
184	26
185	16
186	34
187	24
188	63
189	12
190	8
191	2
192	43
193	7
194	7
195	30
196	3
197	3
198	3
199	3
200	3
201	1
202	17
204	2
205	8
207	9
208	1
209	5
210	32
211	5
212	1
213	62
214	1
215	6
216	7
217	38
218	14
220	3
221	8
222	4
223	27
224	44
226	4
227	5
228	5
233	1
236	7
237	13
239	4
240	5
241	4
243	2
245	3
248	5
251	6
253	64
254	2
258	6
260	5
261	2
263	13
264	3
265	2
266	10
268	12
270	1
271	2
272	2
274	3
275	2
276	1
277	2
278	1
279	1
280	3
281	1
282	1
283	4
284	5
285	3
286	4
287	9
288	8
289	18
290	31
291	19
292	39
