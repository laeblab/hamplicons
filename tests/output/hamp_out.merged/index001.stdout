[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     tests/data/A1gRNA9_S97_L001_R1_001.fastq.gz
[FLASH]     tests/data/A1gRNA9_S97_L001_R2_001.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./hamp_out.merged/index001.extendedFrags.fastq.gz
[FLASH]     ./hamp_out.merged/index001.notCombined_1.fastq.gz
[FLASH]     ./hamp_out.merged/index001.notCombined_2.fastq.gz
[FLASH]     ./hamp_out.merged/index001.hist
[FLASH]     ./hamp_out.merged/index001.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           151
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow "outie" pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 50000 read pairs
[FLASH] Processed 75000 read pairs
[FLASH] Processed 100000 read pairs
[FLASH] Processed 125000 read pairs
[FLASH] Processed 150000 read pairs
[FLASH] Processed 175000 read pairs
[FLASH] Processed 200000 read pairs
[FLASH] Processed 225000 read pairs
[FLASH] Processed 250000 read pairs
[FLASH] Processed 275000 read pairs
[FLASH] Processed 280033 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      280033
[FLASH]     Combined pairs:   110465
[FLASH]     Uncombined pairs: 169568
[FLASH]     Percent combined: 39.45%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 6.815 seconds elapsed
